src/foam/support/support.js

```diff
@@ -1,4 +1,5 @@
 FOAM_FILES([
   //Place support files here.
   // example: { name: 'foam/support/Test' }
+  { name: "foam/support/model/Ticket" }
```

Comment:
Does this really need its own FOAM_FILES?  Can we just add it to the nano.js files?

Especially if we're adding it to the menus, we should make sure the models are loaded when nanos is.